NEWS.md

```diff
@@ -20,14 +21,30 @@ OpenSSL Releases
  - [OpenSSL 1.0.0](#openssl-100)
  - [OpenSSL 0.9.x](#openssl-09x)
 
-OpenSSL 3.5
+OpenSSL 3.6
 -----------
 
-### Major changes between OpenSSL 3.5 and OpenSSL 3.6 [under development]
+### Major changes between OpenSSL 3.5 and OpenSSL 3.6 [02 Sep 2025]
```

Comment:
Please revert this change. It is being done automatically by the release scripts.